apps/web/components/OrderAgain/OrderAgain.vue

```diff
@@ -0,0 +1,195 @@
+                  <span>{{ t('account.ordersAndReturns.orderAgain.priceUp') }}</span>
+                </UiTag>
+                <UiTag
+                  v-else-if="orderGetters.getOrderAgainInformationPrice(item) < orderGetters.getItemPrice(item)"
+                  variant="positive"
+                  size="sm"
+                  class="!font-medium"
+                >
+                  <SfIconArrowDownward size="xs" />
+                  <span>{{ t('account.ordersAndReturns.orderAgain.priceDown') }}</span>
+                </UiTag>
+              </div>
+            </div>
+          </div>
+        </div>
+        <div class="w-full" v-else>
+          <SkeletonsOrderAgainItem class="mt-6" v-for="item in orderGetters.getItems(order)" :key="item.id" />
+        </div>
+      </div>
+      <div class="h-auto flex-shrink-0 flex gap-2 w-full relative mt-3">
+        <div class="typography-text-xs flex gap-1 mr-auto">
+          <span>{{ t('asterisk') }}</span>
+          <span v-if="showNetPrices">{{ t('itemExclVAT') }}</span>
+          <span v-else>{{ t('itemInclVAT') }}</span>
+          <span>{{ t('excludedShipping') }}</span>
+        </div>
+        <SfButton
+          data-testid="quick-checkout-cart-button"
+          @click="addToCart"
+          :disabled="loading || loadingAddToCart"
+          size="lg"
+          variant="secondary"
+          class="ml-auto"
+        >
+          <SfLoaderCircular v-if="loadingAddToCart" class="flex justify-center items-center" size="sm" />
+          <span v-else>{{ t('account.ordersAndReturns.orderAgain.addToCart') }}</span>
+        </SfButton>
+
+        <SfButton data-testid="quick-checkout-checkout-button" @click="close()" size="lg">
+          {{ t('account.ordersAndReturns.orderAgain.cancel') }}
+        </SfButton>
+      </div>
+    </div>
+  </UiModal>
+</template>
+
+<script setup lang="ts">
+import {
+  SfButton,
+  SfIconClose,
+  SfLoaderCircular,
+  SfIconError,
+  SfIconArrowUpward,
+  SfIconArrowDownward,
+  SfListItem,
+} from '@storefront-ui/vue';
+import type { OrderAgainProps } from './types';
+import { orderGetters } from '@plentymarkets/shop-api';
+
+defineProps<OrderAgainProps>();
+const { send } = useNotification();
+const { addModernImageExtension } = useModernImage();
+const { isOpen, addOrderToCart, loading, hasItemsChanged } = useOrderAgain();
+const { t } = useI18n();
+const runtimeConfig = useRuntimeConfig();
+const showNetPrices = runtimeConfig.public.showNetPrices;
+const localePath = useLocalePath();
+const loadingAddToCart = ref(false);
+
+const close = () => {
+  isOpen.value = false;
+};
+
+const goToPage = (path: string) => {
+  close();
+  navigateTo(localePath(path));
+};
+
+const addToCart = async () => {
+  loadingAddToCart.value = true;
+  if (await addOrderToCart()) {
+    send({
+      type: 'positive',
+      message: t('addedToCart'),
+    });
+    goToPage(paths.cart);
+  }
+  loadingAddToCart.value = false;
+};
+</script>
+
+<style scoped>
```

Comment:
Do you think these are reusable? Does it make sense moving into a class here:
```apps/web/assets/style.scss```

Reply:
Could be something we can use more often in the future, I will move it.